app/controllers/laundry_rooms_controller.rb

```diff
@@ -25,4 +25,11 @@ def create_booking
       redirect_to root_path
     end
   end
+
+  def delete_booking
+    active_bookings = current_user.bookings.detect { |booking| booking.time == params[:time] }
+    active_bookings.delete
+    redirect_to root_path
+    flash[:notice] = 'You have successfully canceled your booking.'
```

Comment:
You have to set the flash before the redirect

Reply:
Ok